### DISCUSSION TITLE
Dashboard like template

### DISCUSSION THREAD

**Discussion by harisbal** [reflex-dev/reflex#2141] [Ideas]
Hello Reflex!
Could you please upload a template for a very simple dashboard like web app? Having access to a simple template with a navbar, a sidebar (for multi-page navigation) and basic sign in process would greatly accelerate adoption!
Thank you for this amazing framework, it's been long missing.

**Comment by picklelo** [comment]
@harisbal We now offer a "sidebar" template: https://sidebar-template.reflex.run/dashboard/. It provides basic multi-page functionality out of the box. 

Now running `reflex init` prompts you to select your initial template. We're aiming to add more templates in the future, as well as get the community involved.

**Comment by harisbal** [comment]
Hello @picklelo . This is golden, thank you very much!
